packages/@react-aria/interactions/test/usePress.test.js

```diff
@@ -16,7 +16,7 @@ import {usePress} from '../';
 
 function Example(props) {
   let {elementType: ElementType = 'div', ...otherProps} = props;
-  let {pressProps} = usePress(props);
+  let {pressProps} = usePress(otherProps);
```

Comment:
If we don't use otherProps, then elementType makes it through usePress through as ...domProps and applied through ...pressProps here

Reply:
should pressProps run filterDomProps on what is passed through to usePress instead?

Reply:
well... filterDOMProps just filters public DOM props but I think we use many more than that.